### DISCUSSION TITLE
Outlet is not rendered if no children in the router

### DISCUSSION THREAD

**Discussion by kresli** [TanStack/router#111] [Q&A]
As per documentation, it's not required to use the Outlet element but I can't make it work. Am I  doing something wrong here?
https://codesandbox.io/s/musing-ellis-q3hid?file=/src/index.tsx

**Comment by Codpoe** [comment]
I started a PR, hoping to solve the problem. #113

**Reply by jabidof** [reply]
Hi, 
is there any update on this?

I noticed some routes are not rendered in the case of e.g.:
1. `https://<some URL.url>/verifyMyTokenAndLogMe?adtok=VYgYfk2qoII.....<anyToken>.....6vr4soNTs3NdhFk5RbibvEPzsi6sFgnsbZ`
2. in a component, verify the token, if Ok then `return <Navigate to="/" />` if not Ok `return <Navigate to="/login" />`
3. In the router, the "/" route is defined to return the `element: <Navigate to="/info" />` but for some reason it'll not.

Let me know how to help and if this has already been identified.
